contents/handbook/growth/developer-relations.mdx

```diff
@@ -0,0 +1,232 @@
+|             User demos              |       Developer Experience        |
+|        Community Management         | Developer Experience, Engineering |
+|           User Interviews           |       Product, Engineering        |
+
+
+As we can see, a lot of the roles that are linked to "Developer Relations" are actually performed by "other teams" - and that's not only fine, it's encouraged. 
+
+Ultimately, for a dev-focused company, Developer Relations is bound to be a responsibility of everyone - albeit to largely varying degrees.
+
+At PostHog, we also have [engineers throughout the company](/handbook/company/values#everyone-codes) - our CEO, Lead Designer, and Marketing Lead are all developers. This means that these responsibilities can comfortably be shared around teams instead of falling specifically under a Developer Relations team.
+
+### Names and stuff...
+
+However, as we grow, a DevRel team might make sense for us. As such, it's useful to think about some of the roles that would fit into this team.
+
+To do so, let's borrow a definition from a [blog post from Slack](https://slack.engineering/defining-a-career-path-for-developer-relations/), which says:
+
+*"Developer Relations (DevRel) is an interdisciplinary role that sits in a border space between product, engineering, and marketing."*
+
+This definition may not be the exact truth, but it gives us a framework to conceptualize how some common DevRel roles might fit *at PostHog*:
+
+![Developer Relations Matrix](../../images/handbook/devrel-matrix.png)
+
+Here, Engineering was added just for context, and there are also some notable omissions, such as Developer Educator, Developer Community Manager, Technical Writer, among others.
+
+But let's make sense of this diagram:
+
+#### Axes
+
+On the Y-Axis we have "Product" at the top and "Periphery" at the bottom. "Periphery" encompasses everything that isn't necessarily the *core* product but is also a key component of our offering. This includes stuff like libraries, plugins, integrations, etc.
+
+On the X-Axis we have "Engineering" on the left, which is rather self-explanatory: it encompasses the actual process of building things. On the other hand, we have "Marketing / Non-Engineering", which is an extension to the Slack definition to encompass non-Engineering tasks that also aren't Marketing, such as support, demos, and documentation.
+
+#### Roles
+
+##### Engineering
+
+Pretty straight forward - the team building our core product.
+
+##### Developer Experience
+
+This will vary a lot based on the type of company. If a company's core product is a dev tool, the line between UX and DX becomes increasingly blurry (because all users are developers!), and so Developer Experience might go deeper into product than it does at PostHog, for example. 
+
+For us, DX relates to improving the end-to-end experience for an engineer using (and/or building) our product. More on this later.
+
+##### Developer Advocacy and Evangelism
```

Comment:
As I'm reading, I'm wondering if we do small teams #1067 , then we'd probably have dev relation people as part of the small teams (ie to really nail SDKs _or_ the website/docs), and perhaps dev advocacy would be part of a centralized growth function

Reply:
Yes I believe the Dev Advocates/Evangelists would probably end up falling under a centralized Growth or Marketing function. Unless we end up building a DevRel function too. But given our Marketing is led by a developer, that already borders on DevRel